Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "ipc-client"
+name = "ipc-agent"
```

Comment:
Good catch :)

Reply:
I named my packages with underscores following the convention in `ref-fvm`, and the `builtin-actors`, although apparently not in `rust-libp2p`. I thought it might convert them to underscores, but no, when using it in cargo it would be eg. `cargo test -p ipc-agent`, and `cargo test -p ipc_ipld_resolver`. 

Maybe we could all stick to one?